parsec-evidence-cca.md

```diff
@@ -0,0 +1,71 @@
+# CCA Parsec Evidence
+
+This document describes the format of the credential produced by the Parsec
+service as a key attestation credential for applications running in an Arm CCA realm.
+
+The Parsec service produces a blob of binary data as a response to a key
+attestation operation issued by a client. The service takes in a nonce and uses
+it as freshness for the attestation token.
+
+The attestation token produced by Parsec is a CBOR encoded structure, namely a
+[Combined Attestation Bundle
+(CAB)](https://datatracker.ietf.org/doc/draft-bft-rats-kat/). The format is 
+described below in CDDL.
+
+```
+parsecCcaAttestation = {
+    "kat" => UCCS-KAT,
+    "pat" => CCA-token
```

Comment:
For clarity sake and to avoid confusion I suggest we rename this to 
"ccat" => CCA-token

The reason been the name pat is strikingly similar to PAT (Platform Attestation Token) which has a different implications as per CCA Spec.

Reply:
Hmm, I think the reason for this naming is twofold - to be consistent with the TPM token, and to be consistent with the [KAT draft spec](https://www.ietf.org/archive/id/draft-bft-rats-kat-00.html#name-kat-pat-bundle). Are you refering here to the platform token section in the CCA attestation token? If so, I can try to make it clearer that this claim contains the whole token, not just that part.

Reply:
I agree with Ionut that consistency is the driving factor here, even at the cost of some potential (but in any case very limited) confusion.

Reply:
The confusion is also due to the following statement below, which also needs clarity:

`kat`: A sidecar token used to link the identity key of the application to the Platform Attestation Token (PAT) contained in `pat`. 

The above statement implies that there is PAT inside a `pat`

Reply:
And by the way it is NOT a limited confusion but a serious confusion unless we clarify it adequately, and make is un-ambiguous to readers coming from CCA World!

Reply:
@ionut-arm I understand your point on consistency with the TPM token and the specification.

We can resolve this with adequate clarity that:

The term `pat` and PAT referred in this document refers to entire CCA Token ( and CCA System) and is not tbe confused with the same term used in the CCA specification with a different scope.

Reply:
Yeah, agree, this seems to be down to the lack of clarity in my writing rather than anything else.. Will update the doc to improve on that.